client/state/ui/test/selectors.js

```diff
@@ -30,15 +30,15 @@ describe( 'selectors', () => {
 			const selected = getSelectedSite( {
 				sites: {
 					items: {
-						2916284: { ID: 2916284, name: 'WordPress.com Example Blog' }
+						2916284: { ID: 2916284, name: 'WordPress.com Example Blog', URL: 'https://example.com' }
 					}
 				},
 				ui: {
 					selectedSiteId: 2916284
 				}
 			} );
 
-			expect( selected ).to.eql( { ID: 2916284, name: 'WordPress.com Example Blog' } );
+			expect( selected ).to.contain( { ID: 2916284, name: 'WordPress.com Example Blog' } );
```

Comment:
Should we just update the asserted comparison object to match the expectation (assuming it adds a few computed properties)?